src/addrman.h

```diff
@@ -554,6 +571,28 @@ class CAddrMan
         Check();
     }
 
+    //! See if any to-be-evicted tried table entries have been tested and if so resolve the collisions.
+    void ResolveCollisions(){
+         {
+            LOCK(cs);
+            Check();
+            ResolveCollisions_();
+            Check();
+        }
+    }
+
+    //! Randomly select an address in tried that another address is attempting to evict.
+    CAddrInfo SelectTriedCollision(){
+        CAddrInfo ret;
+        {
```

Comment:
Same Q on `{}`. The lock will go out of scope before `ret`, but I don't think it will matter here as the lock will go out of scope on `return` regardless.

Reply:
I'm using the pattern from select. Since they essentially do the same thing they should look the same.
https://github.com/bitcoin/bitcoin/blob/master/src/addrman.h#L562